pkg/cmd/operator/cleanupjob.go

```diff
@@ -0,0 +1,134 @@
+// Copyright (c) 2023 ScyllaDB.
+
+package operator
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/scylladb/scylla-operator/pkg/controllerhelpers"
+	"github.com/scylladb/scylla-operator/pkg/genericclioptions"
+	"github.com/scylladb/scylla-operator/pkg/helpers"
+	"github.com/scylladb/scylla-operator/pkg/scyllaclient"
+	"github.com/scylladb/scylla-operator/pkg/signals"
+	"github.com/scylladb/scylla-operator/pkg/version"
+	"github.com/spf13/cobra"
+	apierrors "k8s.io/apimachinery/pkg/util/errors"
+	cliflag "k8s.io/component-base/cli/flag"
+	"k8s.io/klog/v2"
+)
+
+type CleanupJobOptions struct {
+	AuthToken   string
+	NodeAddress string
+
+	scyllaClient *scyllaclient.Client
+}
+
+func NewCleanupJobOptions(streams genericclioptions.IOStreams) *CleanupJobOptions {
+	return &CleanupJobOptions{}
+}
+
+func NewCleanupJobCmd(streams genericclioptions.IOStreams) *cobra.Command {
+	o := NewCleanupJobOptions(streams)
+
+	cmd := &cobra.Command{
+		Use:   "cleanup-job",
+		Short: "Runs a cleanup procedure against a node.",
+		Long:  "Runs a cleanup procedure against a node.",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			err := o.Validate()
+			if err != nil {
+				return err
+			}
+
+			err = o.Complete()
+			if err != nil {
+				return err
+			}
+
+			err = o.Run(streams, cmd)
+			if err != nil {
+				return err
+			}
+
+			return nil
+		},
+
+		SilenceErrors: true,
+		SilenceUsage:  true,
+	}
+
+	cmd.Flags().StringVarP(&o.AuthToken, "auth-token", "", o.AuthToken, "Path to a file containing auth token.")
```

Comment:
```suggestion
	cmd.Flags().StringVarP(&o.AuthToken, "auth-token-path", "", o.AuthToken, "Path to a file containing auth token.")
```
using `-path` will disambiguate it from passing token string as value

Reply:
hm, it's more of a `manager-auth-config-path` but either is fine with me

Reply:
done